pkg/resources/network/network.go

```diff
@@ -15,7 +15,17 @@ import (
 // NamespaceName contains resources related to networking.
 const NamespaceName resource.Namespace = "network"
 
+// ConfigNamespaceName contains umerged resources related to networking generate from the configuration.
+//
+// Resources in the ConfigNamespaceName namespace are merged to produce final versions in the NamespaceName namespace.
+const ConfigNamespaceName resource.Namespace = "network-config"
```

Comment:
Just an idea: We could have a `configuration` namespace that is common for any resource type that is configurable. Then something like `talosctl get all -n configuration` would be a dump of all user-supplied things.

Reply:
I'll put this as an issue, as it's easy to rename (just a constant here), and we can come up with good names for `unmerged` configuration and `merged` configuration namespaces, not only for the network.